libbeat/_meta/config.reference.yml

```diff
@@ -610,9 +610,12 @@ output.elasticsearch:
 #setup.template.enabled: true
 
 # Template name. By default the template name is beatname.
-# The version of the beat will always be appended to the given name
-# so the final name is beatname-%{[beat.version]}.
-#setup.template.name: "beatname"
+# The f
+#setup.template.name: "beatname-%{[beat.version]}"
+
+# Template patttern. By default the template patter is "-%{[beat.version]}-*" to apply to the default index settings.
+# The first part is the version of the beat and then -* is used to match all daily indicies.
+#setup.template.pattern: "beatname-%{[beat.version]}-*"
```

Comment:
Not sure I understand this suggestion. You mean something lie `beatname-%{[beat.version]}*`?